src/coreComponents/mesh/generators/InternalWellGenerator.cpp

```diff
@@ -233,14 +241,14 @@ void InternalWellGenerator::constructPolylineNodeToSegmentMap()
 
     // various checks and warnings on the segment and element length
     GEOS_THROW_IF( segmentLength < m_minSegmentLength,
-                   "Error in the topology of well '" << getName() <<
-                   "': we detected a polyline segment measuring less than " << m_minSegmentLength << "m. \n" <<
+                   getDataContext() << ": Error in the topology of the well. We detected a" <<
+                   " polyline segment measuring less than " << m_minSegmentLength << "m. \n" <<
                    "You can change the minimum segment length using the field " << viewKeyStruct::minSegmentLengthString(),
                    InputError );
 
     GEOS_THROW_IF( m_polyNodeCoords[ipolyNode_a][2] < m_polyNodeCoords[ipolyNode_b][2],
-                   "Error in the topology of well '" << getName() <<
-                   "': in the polyline, each segment must be going down. \n" <<
+                   getDataContext() << ": Error in the topology of the well. Wn the polyline"<<
```

Comment:
What is Wn ? When ?

Reply:
I think it is a typo and the correct word was "in"